### ISSUE TITLE
Any idea when jupyterlab be stable enough for regular use?

### ISSUE THREAD

**Issue by ajinkyakolhe112** [jupyterlab/jupyterlab#3021]

**Comment by mjclarke94** [comment]
The Beta milestone is supposed to achieve this and is currently scheduled for the 30th of September (possibly slightly optimistic). The 1.0 release is probably more a concern for developers than end users.

**Comment by blink1073** [comment]
Yep, tracked as [beta](https://github.com/jupyterlab/jupyterlab/milestone/1) and [1.0](https://github.com/jupyterlab/jupyterlab/milestone/2) milestones.

**Comment by ajinkyakolhe112** [comment]
Awesome.... So beta will definitely be good enough to use. Looking forward to using and teaching jupyter lab now. :)
